app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+
+import javax.management.RuntimeErrorException;
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private TaskRepository taskRepository;
```

Comment:
```suggestion
    private final TaskRepository taskRepository;
```

한번 DI로 들어온 값은 `final` 로 불변을 강제하는 것이 좋아요!